Makefile

```diff
@@ -1,15 +1,18 @@
 PROJECT_ROOT=github.com/jaegertracing/jaeger
 # TOP_PKGS is used with 'go test'
-# TODO: try to do this without glide, since it may not be installed initially
-TOP_PKGS := $(shell glide novendor | \
+TOP_PKGS := $(shell go list ./... | \
```

Comment:
so:
```
$ time go list ./...
...
go list ./...  1.51s user 2.29s system 105% cpu 3.595 total
```
Can we do without it altogether? Do we still need TOP_PKGS? If we do, can we use something lighter than `go list`?